Makefile.am

```diff
@@ -37,6 +37,7 @@ swupd_make_pack_SOURCES = \
 	src/config.c \
 	src/delta.c \
 	src/globals.c \
+	src/groups.c \
```

Comment:
I don't think swupd_make_pack needs this source file, since swupd_create_update is the only caller of `write_manifest_plain()`.